src/UCTSearch.cpp

```diff
@@ -392,11 +396,14 @@ int UCTSearch::think(int color, passflag_t passflag) {
     Time elapsed;
     int elapsed_centis = Time::timediff_centis(start, elapsed);
     if (elapsed_centis > 0) {
-        myprintf("%d visits, %d nodes, %d playouts, %d n/s\n\n",
+        auto nn_hr2 = NNCache::get_NNCache()->hit_rate();
+        NNCache::get_NNCache()->dump_stats();
```

Comment:
I'd propose to dump_stats when in DEBUG mode and keep the original output otherwise (without nncache stats)